python/ray/rllib/dqn/common/atari_wrappers.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
+from collections import deque
+import gym
+from gym import spaces
+import cv2
+cv2.ocl.setUseOpenCL(False)
```

Comment:
what does this do?

Reply:
I have no idea -- this is copied from the baselines copy.